small_utils/congestion.cpp

```diff
@@ -191,16 +188,16 @@ CongestionUI::exit_status_t CongestionUI::run(int argc, char** argv)
     dm_get_device_id(_mf, &devID, &hwDevID, &hwChipRev);
     if (!isDeviceSupported(devID))
     {
-    _errorMsg = "mstcongestion is supported for CX4+ devices.";
-    return EXIT_STATUS_DEV_ID_ERR;
+        _errorMsg = "mstcongestion is supported for CX4+ devices.";
+        return EXIT_STATUS_DEV_ID_ERR;
     }
 
     struct tools_open_mcam mcam;
     memset(&mcam, 0, sizeof(mcam));
     reg_access_status_t status = reg_access_mcam(_mf, REG_ACCESS_METHOD_GET, &mcam);
     if (status) {
-        _errorMsg = "Failed to get device capabilities!";
-        return EXIT_STATUS_ERROR;
+        _errorMsg = "mstcongestion is not supported on this device (Failed to get device capabilities).";
+        return EXIT_STATUS_DEV_ID_ERR;
```

Comment:
Is dev id err the best error code for it?
It's not exactly devid issue, but a failure in getting answer of an access register.